File: lecture_012(Sorting)/l_001_normalsort.java
import java.util.Scanner;
public class l_001_normalsort {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        int n = scn.nextInt();
        int[] arr = new int[n];
        input(arr);
        increasingOrder(arr);
        decreasingOrder(arr);
    }
    public static void display(int[] arr) {
        for(int r: arr) {
            System.out.print(r+" ");
        }
        System.out.println();
    }
    public static void input(int[] arr) {
        for(int i=0; i<arr.length; i++) {
            arr[i] = scn.nextInt();
        }
    }
    public static void increasingOrder(int[] arr) {
        for(int i=0; i<arr.length-1; i++) {
            for(int j=i+1; j<arr.length; j++) {
                if(arr[i]>arr[j]) {
                    int temp = arr[i];
                    arr[i] = arr[j];
                    arr[j] = temp;
                }
            }
        }
        display(arr);
    }
    public static void decreasingOrder(int[] arr) {
        for(int i=0; i<arr.length-1; i++) {
            for(int j=i+1; j<arr.length; j++) {
                if(arr[i]<arr[j]) {
                    int temp = arr[i];
                    arr[i] = arr[j];
                    arr[j] = temp;
                }
            }
        }
        display(arr);
    }
}

File: lecture_012(Sorting)/l_001_normalsort.cpp
#include<iostream>
#include<vector>
using namespace std;

void display(vector<int> &arr) {
    for (int r:arr) {        
        cout<<r<<" ";         
    }
    cout<<endl;
}
void input(vector<int> &arr) {   
    for(int i=0; i<arr.size(); i++) {   
        cin>>arr[i];
    }
    display(arr);
}
void increasingorder(vector<int> &arr) {
    for(int i=0; i<arr.size()-1; i++) {
        for(int j=i+1; j<arr.size(); j++) {
            if(arr[i]>arr[j]) {
                int temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }
    display(arr);
}
void decreasingorder(vector<int> &arr) {
    for(int i=0; i<arr.size()-1; i++) {
        for(int j=i+1; j<arr.size(); j++) {
            if(arr[i]<arr[j]) {
                int temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }
    display(arr);
}

int main(int args, char**argv) {
    int n;
    cin>>n;
    vector<int>arr(n,0);   
    input(arr); 
    increasingorder(arr);
    decreasingorder(arr); 

    return 0;                   
}
